docs/dorgeclipseopenj9criuimmutableenvvars.md

```diff
@@ -0,0 +1,57 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -Dorg.eclipse.openj9.criu.ImmutableEnvVars
+
+This system property specifies the environment variables that the VM can detect on restore.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:** This option takes effect only when the `-XX:+EnableCRIUSupport` option is enabled. This option can be used only on restore.
+
+## Syntax
+
+        -Dorg.eclipse.openj9.criu.ImmutableEnvVars=<environment_variables>
+
+Where `<environment_variables>` is a comma-separated list of environment variables.
+
+The following environment variables are included in the list by default and cannot be removed:
+
+- `LANG`
+- `LC_ALL`
+- `LC_CTYPE`
+
+## Explanation
+
+The environment variables are a property of the operating system environment and not the VM. The VM considers these properties as immutable and does not allow the environment variables to change values between checkpoint and restore. Even though the environment variables are always immutable, but with the CRIU support, you can take a checkpoint in one environment and restore in another.
+
```

Comment:
Maybe you meant "are always immutable from a JDK perspective"  or something equivalent ?

Reply:
@tajila @vijaysun-omr - Not sure about this comment. Any suggestions?

Reply:
I think you can remove "Even though the environment variables are always immutable, but with the CRIU support, you can take a checkpoint in one environment and restore in another."